README.md

```diff
@@ -51,7 +51,11 @@ return [
     'components' => [
         'mongodb' => [
             'class' => '\yii\mongodb\Connection',
-            'dsn' => 'mongodb://developer:password@localhost:27017/mydatabase',
+            'dsn' => 'mongodb://@localhost:27017/mydatabase',
```

Comment:
are you sure the `@` should be here?

Reply:
Yes. Im pretty sure. I've just tested it and everything works fine. Feel free to test it for yourself though.